oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/CurrentLocationInformation/CurrentLocationInformation.jsx

```diff
@@ -26,14 +26,15 @@ const NewChildItemMessage = ({ newChildItemParentId }) => {
   const {
     values: { id },
   } = useFormikContext();
+  // not possible to use apiClient because I am working just with a information
```

Comment:
why not calling read -> readDraft(draftLinks) action instead? DraftLinks could then be just object like `{self: /api/vocabularies/.... }`. Also related to: https://github.com/oarepo/oarepo-ui/pull/73#discussion_r1324292809

Reply:
@mirekys Thanks Miro there is a small issue here actually. As our readDraft is like this (and most of our methods actually):

```
readDraft = async (draftLinks) => {
    return this._createResponse(() => {
      const response = this.axiosWithConfig.get(new URL(draftLinks.self).pathname)
      return this.recordSerializer.deserialize(response)
    });
  };
```

We are using the new URL to strip the protocol, IP address and port number. The issue with this is when the link is actually relative e.g. like in this case /api/vocabularies/ the call to new URL fails Failed to construct 'URL': Invalid URL. This could generally be an issue, if for example, for some reason we now start getting links within a record in relative form, all API calls would fail. What do you think would be the most reasonable way to handle this please? I guess in the client.js, we could probably check for each method, if link is absolute, if yes strip, if not just use the relative URL untouched. Or do you think that is too ugly?